.github/scripts/webhook-pr-comment.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+module.exports = () => {
+  try {
+    const outputsFile = fs.readFileSync('cdk-infra/outputs.json').toString();
+    const outputs = JSON.parse(outputsFile);
+
+    const webhook = Object.values(outputs[`auto-builder-stack-enhancedApp-stg-${process.env.GIT_BRANCH}-webhooks`])[0];
```

Comment:
the value for the `auto-builder-stack-enhancedApp-stg-${process.env.GIT_BRANCH}-webhooks` property contains an object that looks something like:

``` json
 {
  "AS12FWebhookEWE312": "url"
 }
```

And that key is different, so it's not really easy to predict how to index it, so I just get the first value in the array, since there should only be one.